docs/source/science/NISAR/Simulated_NISAR.ipynb

```diff
@@ -0,0 +1,11846 @@
+{
```

Comment:
<p>We should be able to drop the previous code cell that assigns <code>creds</code> and in this code cell, simply do this:</p>

<pre class="ql-syntax" spellcheck="false">fs_s3 = earthaccess.__store__.get_s3fs_session(endpoint=asf_s3)
</pre><p>Although the method name is <code>get_s3fs_session</code>, which is a bit misleading, it returns an instance of <code>s3fs.S3FileSystem</code> (not any sort of session instance).   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Ohhh I see now - thanks, Chuck! I'll get that updated!</p>